styleguide/source/assets/scss/05-pages/_people-listing.scss

```diff
@@ -1,7 +1,4 @@
 .ama__people-listing {
-  .container {
```

Comment:
Why was this removed? The menu seems to work fine without this removal

Reply:
@designerneil The `overflow: hidden;` attribute was causing the menu to not show. I was only able to have it expanded by removing the attribute. We can leave the the `.container` there, I don't think it will affect the menu. 

Also, is there a better way than removing `overflow: hidden;`? Seeing if there was a better way, in case of regression issues/bugs.

Reply:
I think the overflow hidden was put in place to prevent the menu from going over the container. I'll check to see if there is a better way.